doc/constants.md

```diff
@@ -3657,7 +3657,7 @@
 - `MER_SWORDMAN09`: 6045
 - `MER_SWORDMAN10`: 6046
 
-### checkquest
+### questprogress
```

Comment:
This file automatically generated. Change it in [db/constants.conf](https://github.com/HerculesWS/Hercules/blob/stable/db/constants.conf#L3759) instead.